index.html

```diff
@@ -39,8 +39,205 @@ <h1 class="hero-title" >I’m Mariane <br>
+                <img class="work-main-img" src="./assets/Snapshoot Portfolio4.svg" alt="">
+                <aside class="work-info">
+                    <h2 class="work-title" >Multi-Post Stories</h2>
+                    <ul class="work-list">
+                        <h4>CANOPY</h4>
+                        <li class="work-list-item" >
+                            <img src="./assets/Counter.svg" alt="">
+                            Back End Dev
+                        </li>
+                        <li class="work-list-item" >
+                            <img src="./assets/Counter.svg" alt="">
+                            2015
+                        </li>
+                    </ul>
+                    <article class="work-description">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    </article>
+                    <ul class="work-tech">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javaScript</li>
+                    </ul>
+                    <article class="work-button ">
+                        <button class="btn" > See project </button>
+                    </article>
+                </aside>
+            </article>
+        </div>
+    </section>
+
+    <section class="hero hero-about ">
+        <div class="container-big">
+            <article class="hero-content separete-items-down">
+                <h1 class="hero-title" >About <br>
+                    Myself <br></h1>
+                <p class="hero-text">
+                    I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+
+                <a href="#">LET'S CONNECT</a>
+
+                <aside class="social-media">
+                    
+                    <a href="">
+                        <img src="./assets/social media icons.svg" alt="">
+                    </a>
+                </aside>
+
+                <article class="work-button work-button-about">
+                    <button class="btn" > Get my resume </button>
+                </article>
+            </article>
+            
+        </div>
+        <article class="languajes">
+            <h2>Languages</h2>
+
+            <ul class="languajes-cont">
+                <li class="languaje-item" >
+                    <img src="./assets/Ellipse 1.svg" alt="">
+                    <p>Javascript</p>
+                </li>
+                <li class="languaje-item" >
+                    <img src="./assets/Ellipse 2.svg" alt="">
+                    <p>HTML</p>
+                </li>
+                <li class="languaje-item" >
+                    <img src="./assets/Ellipse 3.svg" alt="">
+                    <p>CSS</p>
+                </li>
+            </ul>
```

Comment:
- [ ] As suggested by the previous reviewer, kindly consider adding the down arrow next to `Languages` heading because our design should follow the figma design as a project requirement.
- [ ] Also as suggested by the previous reviewer, kindly consider lowering the `font-weight` in `languages`, `frameworks` and `skills`. the root of the problem is that you are giving `font-weight: bold` to `h2` generally. so kindly consider taking h3 or any other tag and style them properly with figma guidelines. in figma these three headings has `font-weight: 500`.
your design
![qw](https://user-images.githubusercontent.com/96683843/219716969-8f9bf9c4-2ccb-4491-b271-4507b157c5c0.jpg)

figma des
![rew](https://user-images.githubusercontent.com/96683843/219716950-90e337ce-f26d-46e3-bde0-ad07e97bbec6.jpg)
ign